docs/user/convert/premade/nyctaxi.rst

```diff
@@ -0,0 +1,74 @@
+NYC Taxi
+========
+
+New York City taxi activity data published by University of Illiois from
+Freedom of Information Law requests to NYC Taxi and Limo Commission.
+More information about the dataset is
+`here <https://publish.illinois.edu/dbwork/open-data/>`__.
+
+U of Illinois hosts the data in a Box web interface and arrives in
```

Comment:
I've got mixed feelings about 'U of...' particularly since I think we only say it twice.